<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>451</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Decoding">
      <attribute name="title">
       <string>Decoding</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>401</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>Do not display SU types</string>
       </property>
       <widget class="QLineEdit" name="lineEditdonotdisplaysus">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>26</y>
          <width>381</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>26 0A</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>401</width>
         <height>121</height>
        </rect>
       </property>
       <property name="title">
        <string>ACARS window</string>
       </property>
       <widget class="QCheckBox" name="checkBoxdropnontextmsgs">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>27</y>
          <width>201</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Drop non text messages</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>25</y>
          <width>48</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Format</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxDisplayformat">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>27</y>
          <width>38</width>
          <height>21</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
       </widget>
       <widget class="QCheckBox" name="checkBoxbeepontextmessage">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>301</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Beep on multi line text message</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkOutputDecodedMessageToUDPPort">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>131</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Output to UDP</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditudpoutputdecodedmessagesaddress">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>90</y>
          <width>231</width>
          <height>21</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>you can output to multiple destinations
by placing a space between destinations.
eg &quot;localhost:12345 localhost:12346&quot;</string>
        </property>
        <property name="text">
         <string>localhost:18765</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>470</y>
         <width>401</width>
         <height>151</height>
        </rect>
       </property>
       <property name="title">
        <string>Audio source</string>
       </property>
       <widget class="QComboBox" name="comboBoxsoundcard">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>381</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxlogwidebandwidthenable">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>321</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Enable widebandwidth (CPU intensive)</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxZMQ">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>ZMQ Audio</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditZmqConnectAddress">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>60</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>60</y>
          <width>61</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Address</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>90</y>
          <width>51</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Topic</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditZmqTopic">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>90</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>401</width>
         <height>71</height>
        </rect>
       </property>
       <property name="title">
        <string>Logging</string>
       </property>
       <widget class="QLineEdit" name="lineEditlogdir">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>201</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxlogenable">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>30</y>
          <width>71</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>26</y>
          <width>81</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Directory</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>390</y>
         <width>401</width>
         <height>71</height>
        </rect>
       </property>
       <property name="title">
        <string>Images</string>
       </property>
       <widget class="QLineEdit" name="lineEditplanesfolder">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>291</width>
          <height>21</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Plane images are stored as {AES}.png or {AES}.jpg in this folder. Additionally if a basestation.sqb database file is present JAERO will use this file for plane lookups.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>26</y>
          <width>81</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Directory</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>401</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>ADS data output</string>
       </property>
       <widget class="QCheckBox" name="checkOutputADSMessageToTCP">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>241</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Enable BaseStation format</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdittcpoutputadsmessagesaddress">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>30</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>localhost:30003</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkTCPAsClient">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>161</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Behave as client</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="logwindowsettings">
      <attribute name="title">
       <string>Log Window</string>
      </attribute>
      <widget class="QLineEdit" name="lineEditplanelookup">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>58</y>
         <width>291</width>
         <height>21</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;URL for plane lookup. {AES} is the plane ICAO number in hex and {REG} is the registration.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>http://www.flightradar24.com/data/airplanes/{REG}</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Plane lookup</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxDisablePlaneLogWindow">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>201</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Disable plane log window</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="voicesettings">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>8</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>Voice Settings</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>401</width>
         <height>191</height>
        </rect>
       </property>
       <property name="title">
        <string>Voice decoding</string>
       </property>
       <widget class="QLabel" name="labelBindAddress">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Socket bind address</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditZMQBind">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>110</y>
          <width>221</width>
          <height>20</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>tcp://*:5550</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="ambeEnabled">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>331</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Local voice decoding in JAERO</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="remoteAmbeEnabled">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>261</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Remote voice decoding</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Topic</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditZMQBindTopic">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>150</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>JAERO</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxZMQ</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBoxsoundcard</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>581</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>641</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxZMQ</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditZmqConnectAddress</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>581</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>580</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxZMQ</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditZmqTopic</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>581</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>610</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>remoteAmbeEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditZMQBind</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>remoteAmbeEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditZMQBindTopic</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxDisablePlaneLogWindow</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditplanelookup</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxZMQ</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBoxlogwidebandwidthenable</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>576</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>635</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
